.github/workflows/sonar.yml

```diff
@@ -24,8 +24,6 @@ jobs:
     runs-on: [ledger-live-4xlarge]
     steps:
       - uses: actions/checkout@v4
-        with:
-          fetch-depth: 0
```

Comment:
Question : by deleting the fetch depth here and in the below Workflows, you will pass to a complete shallow fetch of the remote no ? 
Maybe it should be passed to 1 , so that you limit it to 1 commit from the remote.

Reply:
`fetch-depth` defaults to 1, so that's essentially what I'm doing

Reply:
ok didn't know that

Reply:
thanks